docs/simple-post-request-without-twig.md

```diff
@@ -0,0 +1,109 @@
+# Simple POST request without twig in Controller
+
+If your app is fully SPA (Single Page Application) with REST API, certainly you won't be using twig template for pages (except email).
+
+## Get Ready
+
+Run Symfony's Maker Bundle command `bin/console make:reset-password`. It will generate a lots of useful stuff like Form component, Twig Template, Controller, Entity and Repository.
+
+But now, what we only need is the `ResetPasswordController` and the email template which will be sent to user's inbox after request.
```

Comment:
This makes it sound like the user *should* use Maker, but then modify the controller. But I’m not sure that’s what you meant.

Reply:
Yeah, in this example I use maker then modify the controller. I will modify the paragraphs so it won't sound like _should_ but recommended for starter user.